lib/assessment.md

```diff
@@ -22,14 +22,25 @@ Using this format will make it easier for us to grade you, so that we can get ev
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
 <!-- your answer starts here -->
-
+The web is a service because it is a system of entities requesting and sending information to each other. A wonderful real-world corollary to this is Netflix (the old kind with DVD's). A person (the client) sends a request to Netflix (the server) for a DVD (some information), and Netflix sends that DVD to the client. The web works the same way, with clients (browsers) requesting information from servers (websites) and in turn those servers sending the requested information to the client.
 <!-- your answer ends here -->
 
 ## Question 2
 
 Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body.
 <!-- your answer starts here -->
-
+```HTML
+<html>
```

Comment:
Might want to include the `<!DOCTYPE html>` tag as well; however, the page will generally work without it.